### ISSUE TITLE
Error on startup and when connecting to a device

### ISSUE THREAD

**Issue by beMang** [KatharaFramework/Kathara#287]
### Operating System

Windows 11

### Kathará Version

3.7.4

### Bug Description

When i try to start a lab with lstart, i got this error : 
CRITICAL (APIError) 400 Client Error for
         http+docker://localnpipe/v1.45/containers/2b9662491b9199c6ede343ef46ecb44cbb50f05ea29542d44bec8a3bdae1ca0c/star
         t: Bad Request ("failed to create task for container: failed to create shim task: OCI runtime create failed:
         runc create failed: unable to start container process: error during container init: open
         /proc/sys/net/ipv4/conf/eth0/rp_filter: no such file or directory: unknown")

I go a similar error (wich is related i think) when i try to connect to a device with kathara connect : 
CRITICAL (APIError) 409 Client Error for
         http+docker://localnpipe/v1.45/containers/2b9662491b9199c6ede343ef46ecb44cbb50f05ea29542d44bec8a3bdae1ca0c/exec
         : Conflict ("container 2b9662491b9199c6ede343ef46ecb44cbb50f05ea29542d44bec8a3bdae1ca0c is not running")

Despite the error with lstart, i can see the devices with kathara list. I saw that a similar issue was fixed in release 3.7.4, but i have the same problem.

### Steps To Reproduce

[lab_kathara.zip](https://github.com/KatharaFramework/Kathara/files/15096061/lab_kathara.zip)

Just lanch kathara lstart with the attached lab.

### Expected Behavior

The lab should launch without error and we should be able to connect to the related device.

### Check Command Output

```shell
┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│                                                     System Check                                                     │
└──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
Current Manager is:             Docker (Kathara)
Manager version is:             26.0.0
Python version is:              3.11.7 (tags/v3.11.7:fa7a6f2, Dec  4 2023, 19:24:49) [MSC v.1937 64 bit (AMD64)]
Kathara version is:             3.7.4
Operating System version is:    Windows-10-10.0.22631-SP0
[Deploying devices]   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1/1
[Deleting devices]   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1/1
✓ Container run successfully.
```

**Comment by tcaiazzi** [comment]
Hi @beMang, 

Thanks for opening the issue, and sorry for the inconvenient! Yes, we already fixed the problem, but we missed some corner cases on Windows platforms. However, we are already crafted a solution, and we will release it soon. 

I provided here a temporary version of the package that contains the patch:
- [Kathara-windows-installer-x64-3.7.5.zip](https://github.com/KatharaFramework/Kathara/files/15128524/Kathara-windows-installer-x64-3.7.5.zip)

You can use it while waiting for the release. 

In alternative, since the error is caused by an update of Docker that changes some internals, and it is raised from version `26.0.0` of the engine, you can also downgrade Docker to a previous version. 

Please let me know if this helps!

**Comment by beMang** [comment]
Hi @tcaiazzi,

The problem seems to be solved with this version with the temporary version, I can connect to the device. I haven't tried to downgrade docker, but the fix is working ^^

Thanks you for the fix

**Comment by tcaiazzi** [comment]
Hi @beMang, 

We officially released version 3.7.5!
